06-statistical-testing.Rmd

```diff
@@ -1,5 +1,7 @@
 # Statistical testing {#c06-statistical-testing}
 
+<!-- author review -->
+
 When analyzing results from a survey, the point estimates described in Chapter \@ref(c05-descriptive-analysis) help us understand the data at a high level. Still, researchers and the public often want to make comparisons between different groups. These comparisons are calculated through statistical testing, where we compare the point estimates and variance estimates of each statistic to see if statistically significant differences exist. The general idea of statistical testing is the same for data obtained through surveys and data obtained through other methods, but the importance lies in ensuring the variance is calculated correctly. Functions in the {survey} packages allow for the correct estimation of the variances. This chapter will cover the following statistical tests with survey data and functions:
```

Comment:
I call this "point estimates" in this chapter, but we only refer to them as descriptive statistics in chapter 5.  Thoughts?  Will need to either change it here, or add a reference to point estimates in chapter 5.

Reply:
@szimmer since you are reviewing chapter 5, can we add 'point estimates' and a definition there, please?